lib/linguist/languages.yml

```
@@ -3454,6 +3454,14 @@ Mako:
   tm_scope: text.html.mako
   ace_mode: text
   language_id: 221
+Manifest:
```

```suggestion
JAR Manifest:
```

_"Manifest"_ on its own is vague and could refer to any number of manifest formats.